service/src/main/java/org/cbioportal/service/impl/SampleServiceImpl.java

```diff
@@ -173,11 +174,15 @@ private void processSamples(List<Sample> samples, String projection) {
 
         if (projection.equals("DETAILED")) {
             Map<String, Set<String>> sequencedSampleIdsMap = new HashMap<>();
+            Map<String, Set<String>> fusionSampleIdsMap = new HashMap<>();
```

Comment:
Should we call this structuralVariantSampleIdsMap?

Reply:
good idea. renamed.